src/runtime_src/core/tools/xbutil2/OO_P2P.h

```diff
@@ -27,7 +27,7 @@ class OO_P2P : public OptionOptions {
   OO_P2P( const std::string &_longName, bool _isHidden = false);
 
  private:
-  std::string m_device;
+  std::vector<std::string> m_device;
```

Comment:
The code is now working with a collection of devices instead of a device.   

Please update the variable from `m_device` to `m_devices`.